packages/sdk-router/src/sdk.ts

```diff
@@ -182,17 +457,53 @@ class SynapseSDK {
     destQuery: Query
   ): Promise<PopulatedTransaction> {
     token = handleNativeToken(token)
-    const originRouter: SynapseRouter = this.synapseRouters[originChainId]
-    return originRouter.routerContract.populateTransaction.bridge(
+    let bridgeOriginQuery: {
+      swapAdapter?: string
+      routerAdapter?: string
+      tokenOut: string
+      minAmountOut: BigNumber
+      deadline: BigNumber
+      rawParams: string } = { ...originQuery}
+
+      let bridgeDestQuery: {
+        swapAdapter?: string
+        routerAdapter: string
+        tokenOut: string
+        minAmountOut: BigNumber
+        deadline: BigNumber
+        rawParams: string
+      } = { ...destQuery, routerAdapter: destQuery.swapAdapter}
+
+    let isCCTP = (originRouterAddress.toLowerCase() === this.synapseCCTPRouters[originChainId].routerContract.address.toLowerCase())
+    // Determine which router to use based on the address
+    const originRouter: SynapseRouter | SynapseCCTPRouter = isCCTP ? this.synapseCCTPRouters[originChainId] : this.synapseRouters[originChainId];
+
+    if (isCCTP) {
+      bridgeOriginQuery = {
+        ...originQuery,
+        routerAdapter: originQuery.swapAdapter,
+      };
+      delete bridgeOriginQuery.swapAdapter;
+
+      bridgeDestQuery = {
+        ...destQuery,
+        routerAdapter: destQuery.swapAdapter,
+      };
+      delete bridgeDestQuery.swapAdapter;
+    }
```

Comment:
This feels weird. Is there a way to avoid this by unifying `swapAdapter` and `routerAdapter` fields? These mean exactly the same thing, reason for different naming is that CCTP Router is using the updated RouterV2 naming conventions.

Reply:
This was due to Typescript and Ethers Type blocking the transaction going through. 

I reworked the types fully and unionized them across both CCTP router and synapse router. 

See this: https://github.com/synapsecns/sanguine/pull/1059/files#diff-87c018c5e6200d98efe1ba3e088f0a50c7f2afd4f1b98d0417055de255cf751eR513

Not sure of a better way to handle it at the moment, but I standardized all query getters to respond with the relevant swapAdapter or routerAdapter key depending on if CCTP or not. Should mark as a todo to write some tests for that specifically.